content/en/observability_pipelines/monitoring.md

```diff
@@ -0,0 +1,65 @@
+
+Health statuses are determined by specific metrics based on thresholds and default time windows. The available statuses are as follows:
+
+- `Healthy`: Indicates the Worker is not falling behind.
+- `Warning`: Indicates the Worker is not performing optimally and is at risk of falling behind.
+- `Critical`: Indicates that the Worker is falling behind.
+
+You can also view health, resource utilization, and data delivery graphs to determine the state of your pipelines and components.
+
+Health graphs are available for the following metrics:
+- Events unintentionally dropped
+- Errors
+- Lag time (only available for sources)
+- Utilization
+
+Data delivery graphs are available for the following metrics:
+- Events in/out per second
+- Bytes in/out per second
+
+Resource utilization graphs are available for the following metrics:
+- CPU usage
+- Memory usage
+- Disk usage (only available for destinations)
+
+**Note**: These graphs are only available for users enrolled in the remote configuration private beta.
+
+## See the status of your pipelines and components
+
+1. Navigate to [Observability Pipelines][2].
+1. Click on a pipeline.
+1. Hover over the graphs to see specific data points.
+
+## Pipeline health metrics
+
+| Metric        | OK       | Warning    | Critical  | Description                       |
+| ------------  | :------: | :--------: | :-------: | --------------------------------- |
+| CPU usage     | <= 0.85  | > 0.85     | N/A       | Tracks how much CPU a Worker process is using. <br> A value of `1` indicates that a Worker process does not have any more headroom in the host or compute units running it. This can lead to possible issues such as processing latency going out of bounds, upstream/downstream overload, and so on.|
+| Memory usage  | >= 0.15  | < 0.15     | N/A       | Tracks the amount of used and free memory on the host. The Worker is not memory bound but high memory usage can indicate leaks.
+
+## Component health metrics
+
+| Metric        | Sources   | Transforms| Destinations | OK      | Warning  | Critical  | Description                       |
+| ------------  | :-------: | :-------: | :----------: | :-----: | :------: | :-------: | --------------------------------- |
+| Events dropped  | {{< X >}} | {{< X >}} |{{< X >}}     | ==0     | N/A      | > 0       |Expected to always be `0`. If you configured the Worker to intentionally drop data, for example using the `filter` transform, that data is not counted here. Therefore, a single error indicates that the Worker is not in a healthy state.|
+| Total errors  |{{< X >}}  |{{< X >}}  |{{< X >}}     | ==0     | >0       | N/A       | The total number of errors encountered by the component. Diagnostic Logs provides more information about specific internal error logs. |
```

Comment:
Change description to:

The total number of errors encountered by the component. These errors are also emitted as Diagnostic Logs, which provides more information about specific internal error logs and can be viewed by clicking on each component of a pipeline under "Diagnostic Logs".

Reply:
Yes, linking out to Diagnostic logs is better. I'll add the link to this doc once Troubleshooting is live.